qt/place_page_dialog.cpp

```diff
@@ -137,5 +166,10 @@ PlacePageDialog::PlacePageDialog(QWidget * parent, place_page::Info const & info
   setWindowTitle(ppTitle.c_str());
 }
 
+std::optional<RouteMarkType> PlacePageDialog::GetRoutePointAddMode() const { return m_routePointAddMode; };
+void PlacePageDialog::SetRoutePointAddMode(RouteMarkType routePointAddMode)
+{
+    m_routePointAddMode = routePointAddMode;
```

Comment:
Is your indentation/tab set to 4 spaces instead of 2?
```suggestion
  m_routePointAddMode = routePointAddMode;
```